test/Misc/expression_too_complex.swift

```diff
@@ -1,4 +1,4 @@
-// RUN: %target-parse-verify-swift -solver-memory-threshold 5000
+// RUN: %target-parse-verify-swift -solver-memory-threshold 4000
```

Comment:
What's the intention behind making this change? Does using gold have an effect on `%target-parse-verify-swift`, or was this included by accident?